test/e2e/e2e_ginkgo/utilg_test.go

```diff
@@ -0,0 +1,76 @@
+package e2e_ginkgo
+
+import (
+	"fmt"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/apiserver/pkg/storage/names"
+	"time"
+
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/api/client"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/api/client/clientset/versioned"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/lib/operatorclient"
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	cleaner                    *namespaceCleaner
+	genName                          = names.SimpleNameGenerator.GenerateName
```

Comment:
gofmt your files.

Reply:
go fmt still kept the same formatting. I believe for this one, it aligns on just the type.